### PR TITLE
Backport PR #14414 on branch 3.6.x (Fix CI: remove/update broken docs links)

### PR DESCRIPTION
**Pull request by krassowski**
## References

Backport PR #14414: Fix CI: remove/update broken docs links

### CODE REVIEW

File: packages/services/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Javascript client for the Jupyter services REST APIs
 
-[API Docs](http://jupyterlab.github.io/jupyterlab/)
+[API Docs](https://jupyterlab.readthedocs.io/en/latest/api/)
```
Comment (fcollonval) [inline packages/services/README.md:5]:
```suggestion
[API Docs](https://jupyterlab.readthedocs.io/en/3.6.x/api/)
```

**Review by fcollonval — CHANGES_REQUESTED** [review]
Here is a suggestion to fix the integrity check

File: packages/services/README.md (hunk 1 above)
Comment (krassowski) [reply packages/services/README.md]: Thank you!

**Comment by krassowski** [comment]
The check is passing now. The remaining failures are unrelated - merging.

**Pull request merged**